### PR TITLE
  Added Servinform

### PR DESCRIPTION
**Pull request by viscerae**
Added Servinform

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Present Technologies](https://www.present-technologies.com/) [:rocket:](https://www.present-technologies.com/careers/) | Delivering Quality Software. | `Coimbra` `Porto` <br> `Sertã` |
 | [Radically Digital](https://radically.digital/) [:rocket:](https://radically.digital/careers/) | Software, ideation, strategy and design consultancy. | `Lisboa` |
 | [Reaktor](https://www.reaktor.com) [:rocket:](https://www.reaktor.com/careers) | A global strategy, design, and technology partner. | `Lisboa` |
+| [Servinform](https://www.servinform.es/) [:rocket:](https://www.servinform.es/unete-al-equipo/)| Consultancy and Automation. | `Lisboa` |
```
Comment (vpereira01) [inline README.md:94]:
```suggestion
| [Servinform](https://www.servinform.es/pt-pt/) [:rocket:](https://www.servinform.es/pt-pt/junte-se-a-equipa/)| Consultancy and Automation. | `Lisboa` |
```

Probably this is first company using Spanish as the default language, so I'm bit unsure about the defaults.

I think using the URLs for Portuguese or English is preferable.

**Comment by vpereira01** [comment]
Does the company hire for technical jobs in Lisbon?

From the opening jobs listing there's just one for instructor so I'm not sure this company fits this repo.

**Comment by viscerae** [comment]
> Does the company hire for technical jobs in Lisbon?
> 
> From the opening jobs listing there's just one for instructor so I'm not sure this company fits this repo.

Yes they do it's a new branch in Lisbon so they are still expanding and adding new hiring positions.

File: README.md (hunk 1 above)
Comment (viscerae) [reply README.md:94]: Agreed!

**Review by vpereira01 — APPROVED** [review]

**Review by afonsojramos — APPROVED** [review]
If indeed they do intend to start hiring for tech roles, thanks for the addition!

**Comment by marmelo** [comment]
Thank you @viscerae!

**Pull request merged**